dicomImport/matRad_importDicomRtss.m

```diff
@@ -66,9 +66,11 @@
                              
     structures(i).structNumber = structInfo.ROIContourSequence.(...
                                  listOfContStructs{i}).ReferencedROINumber;
-    structures(i).structColor  = structInfo.ROIContourSequence.(...
-                                 listOfContStructs{i}).ROIDisplayColor;  
-                             
+    if isfield(structInfo.ROIContourSequence.(listOfContStructs{i}),'ROIDisplayColor')
+        structures(i).structColor  = structInfo.ROIContourSequence.(...
+                                     listOfContStructs{i}).ROIDisplayColor;  
+    end
+
```

Comment:
Can this cause problems if structures(i).structColor is not defined? In principle this could happen? Do we need a default value here?